Staging/ProBook6sC200sBiosFanReset/ProBookFanReset.c

```diff
@@ -0,0 +1,304 @@
+/**
+ * \file ProBookFanReset.c
+ * Overwrite embedded controller CPU fan register to automatic mode,
+ * resetting the fan to factory behavior at system startup.
+ *
+ * HP ProBook 4330s, 4530s, and 4730s models only:
+ * running this code on any other laptop might brick the device.
+ */
+
+/*-
+ * Copyright (c) 2012 RehabMan
+ *               2023 Ubihazard
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
```

Comment:
Unfortunately we do not accept GPL code in OpenCore.

Reply:
I see, it makes sense. Unfortunately, @RehabMan released his original code GPL‘d, and this can‘t be changed (and I can‘t relicense it).

I‘ll just keep it in my fork then.

Reply:
Thank you for your hard work!